actions/databases.go

```diff
@@ -0,0 +1,290 @@
+package actions
+
+import (
+	"errors"
+	"log"
+	"strconv"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/gobuffalo/buffalo"
+)
+
+// DatabasesList gets a list of databases for a given account
+//   If the `all=true` parameter is passed it will return a list of clusters in addition to instances.
+func DatabasesList(c buffalo.Context) error {
+	// if all param is given, we'll return information about both instances and clusters
+	// otherwise, only database instances will be returned
+	all := false
+	if b, err := strconv.ParseBool(c.Param("all")); err == nil {
+		all = b
+	}
+
+	rdsClient := RDS[c.Param("account")]
+	var clustersOutput *rds.DescribeDBClustersOutput
+	var instancesOutput *rds.DescribeDBInstancesOutput
+	var err error
+
+	if all {
+		clustersOutput, err = rdsClient.Service.DescribeDBClusters(&rds.DescribeDBClustersInput{})
+		if err != nil {
+			log.Println(err.Error())
+		}
+	}
+
+	instancesOutput, err = rdsClient.Service.DescribeDBInstances(&rds.DescribeDBInstancesInput{})
+	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+			log.Println(aerr.Error())
+			return c.Error(400, aerr)
+		} else {
+			log.Println(err.Error())
+			return err
+		}
+	}
```

Comment:
```golang
if err != nil {
    log.Println(err.Error())
    if aerr, ok := err.(awserr.Error); ok {
        return c.Error(400, aerr)
    }
    return err
}
```

Reply:
For some reason I thought that `aerr.Error()` will print more detailed information than just `err.Error()`, that's why I had all the `if`s, but I just realized they print the exact same error (which  is just a String, and we're doing the type casting just so we can check the specific error type). So I think I can simplify all of those blocks as you suggested.